website/content/docs/agent-and-proxy/agent/process-supervisor.mdx

```diff
@@ -0,0 +1,48 @@
+---
+layout: docs
+page_title: Vault Agent's Process Supervisor Mode
+description: >-
+  Vault Agent's Process Supervisor Mode allows Vault secrets to be injected
+  into a process via environment variables using Consul Template markup.
+---
+
+# Vault Agent' Process Supervisor Mode
+
+Vault Agent's  Process Supervisor Mode allows Vault secrets to be injected into
+a process via environment variables using
+[Consul Template markup][consul-templating-language].
+
+<Tip title="Kubernetes">
+  If you are running your applications in a Kubernetes cluster, we recommend evaluating the
+  [Vault Secrets Operator](/vault/docs/platform/k8s/vso) instead.
+</Tip>
+
+<Warning>
+  Vault Agent's Process Supervisor Mode is in public beta. <br />
+  Please provide your feedback by opening a GitHub issue [here](https://github.com/hashicorp/vault/issues).
+</Warning>
+
+
+## Functionality
```

Comment:
I think we should add a section or paragraph describing the exit behaviour, i.e. that Agent will exit if the supervised process exits, and that if you need to guard against restarts, you can monitor Agent (via e.g. systemd/kubernetes restart policy... etc) the same way you'd monitor your app without Agent

Reply:
see https://github.com/hashicorp/vault/pull/21056/files#diff-f91188481f71481c3b2290caa71d71b8e4fb6d18f816366f8abb5c012ba9ed8bR28-R29 (could probably use some re-wording)

Reply:
Looks good!